lib/CL/pocl_llvm_api.cc

```diff
@@ -385,7 +390,12 @@ int pocl_llvm_build_program(cl_program program,
 
   // the per-file types don't seem to override this 
   la->OpenCLVersion = cl_std_i;
+#ifdef HSA_RUNTIME_IS_ROCM
```

Comment:
Again ROCM-stuff springled to generic parts of pocl. A new device hook needed perhaps?